pkg/deploy/pluginregistry/pluginregistry.go

```diff
@@ -119,3 +123,15 @@ func (p *PluginRegistry) SyncDeployment() (bool, error) {
 	spec := p.GetPluginRegistryDeploymentSpec()
 	return deploy.SyncDeploymentSpecToCluster(p.deployContext, spec, deploy.DefaultDeploymentDiffOpts)
 }
+
+func (p *PluginRegistry) createGatewayConfig() *gateway.TraefikConfig {
+	pathPrefix := "/" + deploy.PluginRegistryName
```

Comment:
`/v3` ?

Reply:
hmm, I don't think we're doing that in current deployments. That would mean that `<che-url>/plugin-registry` return 404, is that ok?

Reply:
Maybe I don't completely understand the gateway config.
But plugins are available under `/v3` subpath
So, up to you

Reply:
ok, I'm not comfortable doing this change as it is not really related to the issue and it's unnecessary risk. So I'm keeping plugin registry reachable on `/plugin-registry`